code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -2627,6 +2627,22 @@ var/list/bloodcult_exitportals = list()
 	plane = ABOVE_HUMAN_PLANE
 	mouse_opacity = 0
 
+// RUNE XXI
+/datum/rune_spell/ward
+	name = "Unholy threeshold"
+	desc = "Create a rune that will alert you if someone tresspass."
```

Comment:
trespasses
(also non-indicative that multiple people are alerted by the rune instead of only one)